rust/Dockerfile

```diff
@@ -1,20 +1,6 @@
-# Global args to use in build commands
 ARG ALPINE_VERSION="3.19"
-ARG CARGO_CHEF_VERSION="0.1.62"
-ARG RUSTUP_VERSION="1.26.0"
-ARG RUSTUP_x86_DOWNLOAD_SHA256="7aa9e2a380a9958fc1fc426a3323209b2c86181c6816640979580f62ff7d48d4"
-ARG RUSTUP_aarch64_DOWNLOAD_SHA256="b1962dfc18e1fd47d01341e6897cace67cddfabf547ef394e8883939bd6e002e"
-ARG RUST_VERSION="1.74.1"
```

Comment:
I suspect if you just update this to our workspace version, CI will be fast again (after a merge to main, since that is where the cache is written)

Reply:
(For historical record, we tried that in #4033)